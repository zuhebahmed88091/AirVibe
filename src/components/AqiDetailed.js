import React from 'react';
import PropTypes from 'prop-types';

const AqiDetailed = ({ dataname, data }) => (
  <div className="data-component-wrap">
    <div className="data-component-name">
      {dataname}
    </div>
    <div className="data-component">
      {data}
    </div>
  </div>
);

AqiDetailed.propTypes = {
  dataname: PropTypes.string.isRequired,
  data: PropTypes.string.isRequired,
};

export default AqiDetailed;
