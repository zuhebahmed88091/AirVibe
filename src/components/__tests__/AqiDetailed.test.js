import React from 'react';
import { render } from '@testing-library/react';
import AqiDetailed from '../AqiDetailed';

describe('AqiDetailed', () => {
  it('should render AqiDetailed component', () => {
    const dataname = 'CO';
    const data = '1';
    render(<AqiDetailed dataname={dataname} data={data} />);
    expect(AqiDetailed).toMatchSnapshot();
  });
});
