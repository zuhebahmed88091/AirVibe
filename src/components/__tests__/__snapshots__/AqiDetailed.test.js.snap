// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AqiDetailed should render AqiDetailed component 1`] = `[Function]`;
